### DISCUSSION TITLE
access to pyAIMS on new singularity distribution

### DISCUSSION THREAD

**Discussion by sarafv** [brainvisa/brainvisa.github.io#95] [Q&A] [answered]
Hello,

I moved to BrainVISA 5.0.4 that I installed with Singularity. How can I access to BrainVISA's python to run a python script involving AIMS?

Thank you

Sara

**Comment by denisri** [comment, marked as answer]
Hi,
Once you have setup an environment as explained in the install instructions, and set the `bin/` directory into your host system `PATH`, you can either open a bash shell in the container (`bv bash`) and run python scripts from it, or run them using `bv python script.py`. The entry point to the container is the `bv` command, which runs singularity with the needed options.
Denis

**Comment by sarafv** [comment]
thank you!, this is working for me . Here an example of my command :

Extracting sulci meshes :

/home/sara.fernandezvidal/epi_projects/epiSTIM_project/software/brainvisa/bin/bv python /home/sara.fernandezvidal/epi_projects/epiSTIM_project/epiSTIM/share/aims_scripts/aims_merge_sulci_mesh.py /home/sara.fernandezvidal/epiloc/brainvisa_databases/newdb/brainvisa_db/patients/CHAm/t1mri/default_acquisition/default_analysis/folds/3.1/LCHAm.arg /tmp/tmpp5wv9l2e/sulci.ply

And after converting ply on vtk : 
<img width="532" alt="image" src="https://user-images.githubusercontent.com/26764989/152137800-44986ce6-4940-4de4-8126-3dbb9c647722.png">
